src/mcp-web-reader/README.md

```
@@ -1,75 +1,139 @@
 
--   `--mode, -m`: `raw | markdown | jina`
--   `--url, -u`: Source URL (required)
--   `--depth, -d`: `basic | advanced` (default: basic). Advanced adds:
-    -   absolute anchors on headings
-    -   Jina-like header (title, URL Source, published time)
-    -   absolute URLs for links/images
--   `--tokens, -T`: Maximum output tokens (approx. GPT-3 tokenizer)
--   `--save-tokens, -s`: Compact code blocks (strip trailing spaces, collapse blank lines) and mild whitespace normalization
--   `--out, -o`: Write to file
+| Engine | Speed | Quality | Requirements |
+|--------|-------|---------|--------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)